Add config file name explanation in README (Close #264) (#266)
meilisearch/docs-scraper: README.md

### Set your Config File

> The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class). The config file will be passed as an argument, so you can use any name you like.

Reviewer: ```suggestion
The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class). The config file will be passed as an argument, so you can name your file whatever you want.
```

> The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class). The config file will be passed as an argument, so you can name your file whatever you want.

Reviewer: ```suggestion
The scraper tool needs a config file to know which content you want to scrape. This is done by providing **selectors** (e.g. the HTML tag/id/class). The config file IS passed as an argument. It follows no naming convention and may be named as you want.
```
